docs/en/developer/create-web-component/VUE.md

```diff
@@ -461,3 +564,26 @@ task cleanUp(type: Delete) {
 
 jar.finalizedBy cleanUp
 ```
+
+## C. Node.js installation
+
+### Mac OS X
```

Comment:
```
#### Mac OS X
```

Reply:
I'm leaving it as is because ### Mac OS X is the logical lower hierarchy after ## C. Node.js installation. Making it #### Mac OS X would skip a hierarchy.